### PR TITLE
Replace monitor value with accuracy

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Resolve https://github.com/optuna/optuna-examples/issues/31. More precisely, in the current example files, fastAI's callback monitor is validation loss by default, however, the study direction is `maximization`. 

## Description of the changes

Use `accuracy` for monitor value and remove `validate` since the validation accuracy is already stored.

### CODE REVIEW

File: fastai/fastaiv1_simple.py [hunk 1]
```diff
@@ -70,11 +70,11 @@ def objective(trial):
         model,
         silent=True,
         metrics=[vision.accuracy],
-        callback_fns=[partial(FastAIV1PruningCallback, trial=trial, monitor="valid_loss")],
+        callback_fns=[partial(FastAIV1PruningCallback, trial=trial, monitor="accuracy")],
     )
     learn.fit(EPOCHS)
 
-    return learn.validate()[-1].item()
+    return learn.recorder.metrics[0][0].item()
```
Comment (hvy) [inline fastai/fastaiv1_simple.py:77]: Please correct me if I'm wrong but my understanding is that metrics are reported by epoch. Are we correctly indexing the last epoch?

**Review by hvy — COMMENTED** [review]
Thanks for the PR!

I'm not that familiar with fastai but it seems reasonable. I left a single comment if you could take a look. Thanks!

File: fastai/fastaiv1_simple.py (hunk 1 above)
Comment (nzw0301) [reply fastai/fastaiv1_simple.py]: Thank you for your comment! Your comment is absolutely right. The first index should be `-1` to return the validation accuracy at the last epoch.

Comment (hvy) [reply fastai/fastaiv1_simple.py]: Thanks for the quick fix!

**Review by hvy — APPROVED** [review]
LGTM!
